packages/devtools_app/lib/src/extensions/extension_screen_controls.dart

```diff
@@ -18,114 +18,140 @@ import '../shared/routing.dart';
+          const SizedBox(width: denseSpacing),
+          Row(
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              RichText(
+                text: GaLinkTextSpan(
+                  link: GaLink(
+                    display: 'Report an issue',
+                    url: ext.issueTrackerLink,
+                    gaScreenName:
+                        gac.DevToolsExtensionEvents.extensionScreenId.name,
+                    gaSelectedItemDescription:
+                        gac.DevToolsExtensionEvents.extensionFeedback(ext),
+                  ),
+                  context: context,
+                ),
+              ),
+              const SizedBox(width: denseSpacing),
+              _ExtensionContextMenuButton(
+                ext: ext,
+                onForceReload: onForceReload,
+              ),
+            ],
           ),
-        ),
-        const SizedBox(width: denseSpacing),
-        ValueListenableBuilder<ExtensionEnabledState>(
-          valueListenable:
-              extensionService.enabledStateListenable(extension.displayName),
-          builder: (context, activationState, _) {
-            if (activationState == ExtensionEnabledState.enabled) {
-              return ContextMenuButton(
-                iconSize: defaultIconSize,
-                buttonWidth: buttonMinWidth,
-                menuChildren: <Widget>[
-                  PointerInterceptor(
-                    child: MenuItemButton(
-                      onPressed: () {
-                        // Do not send analytics here because the user must
-                        // confirm that they want to disable the extension from
-                        // the [DisableExtensionDialog]. Analytics will be sent
-                        // there if they confirm that they'd like to disable the
-                        // extension.
-                        unawaited(
-                          showDialog(
-                            context: context,
-                            builder: (_) =>
-                                DisableExtensionDialog(extension: extension),
-                          ),
-                        );
-                      },
-                      child: const MaterialIconLabel(
-                        label: 'Disable extension',
-                        iconData: Icons.extension_off_outlined,
+        ],
+      ),
+    );
+  }
+}
+
+class _ExtensionContextMenuButton extends StatelessWidget {
+  const _ExtensionContextMenuButton({
+    required this.ext,
+    required this.onForceReload,
+  });
+
+  final DevToolsExtensionConfig ext;
+
+  final VoidCallback onForceReload;
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<ExtensionEnabledState>(
+      valueListenable: extensionService.enabledStateListenable(ext.displayName),
+      builder: (context, activationState, _) {
+        if (activationState == ExtensionEnabledState.enabled) {
```

Comment:
Nit: consider negating this condition and returning `const SizedBox.shrink()` so the returns are visually closer together.